EmployeeAPI/Controllers/DeveloperController.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Employees.Models;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace EmployeeAPI.Controllers
+{
+    /// <summary>
+    ///  Developers Controller
+    /// </summary>
+    [Route("api/v1/developer")]
+    [Produces("application/json")]
+    [ApiController]
+    public class DeveloperController : ControllerBase
```

Comment:
No need to add controller for each position, maybe will need it later on. but for now let's stick to one controller EmployeesController